packages/Webkul/Shop/src/Resources/views/components/layouts/header/mobile/index.blade.php

```diff
@@ -132,10 +132,9 @@ class="max-h-[30px]"
                 aria-label="@lang('shop::app.components.layouts.header.bagisto')"
             >
                 <img
-                    src="{{ bagisto_asset('images/logo.svg') }}"
-                    alt="Bagisto"
-                    width="131"
-                    height="29"
+                    src="{{ core()->getCurrentChannel()->logo_url ?? bagisto_asset('images/logo.svg') }}"
+                    alt="{{ config('app.name') }}"
+                    style="height: 30px"
```

Comment:
There should be no inline style. Width and height are specifically given for some reason, please revert it.